tests/v2/validation/certrotation/cert_rotation_test.go

```diff
@@ -38,10 +42,38 @@ func (r *V2ProvCertRotationTestSuite) SetupSuite() {
 }
 
 func (r *V2ProvCertRotationTestSuite) TestCertRotation() {
-	r.Run("test-cert-rotation", func() {
-		require.NoError(r.T(), rotateCerts(r.client, r.client.RancherConfig.ClusterName))
-		require.NoError(r.T(), rotateCerts(r.client, r.client.RancherConfig.ClusterName))
-	})
+	id, err := clusters.GetV1ProvisioningClusterByName(r.client, r.client.RancherConfig.ClusterName)
+	require.NoError(r.T(), err)
+
+	cluster, err := r.client.Steve.SteveType(ProvisioningSteveResouceType).ByID(id)
```

Comment:
Typo it should be: ProvisioningSteveResourceType

Reply:
good catch, thanks